zebROS_ws/src/behaviors/CMakeLists.txt

```diff
@@ -91,12 +91,15 @@ include_directories(
 #Climb.action
 #Elevator.action
 #Align.action
+#PathIntake.action
```

Comment:
This doesn't need to be added. The entire list here was moved to behavior_actions and never removed from here in the behaviors package. Appreciate trying to keep it consistent, but this is probably just an opportunity to clean it up correctly since you're in there.